api/query.proto

```diff
@@ -56,25 +56,46 @@ option (grpc.gateway.protoc_gen_swagger.options.openapiv2_swagger) = {
 };
 
 message QueryTicketsRequest {
-  // A Pool is consists of a set of Filters.
+  // The Pool representing the set of Filters to be queried.
   Pool pool = 1;
 }
 
 message QueryTicketsResponse {
-  // Tickets that satisfy all the filtering criteria.
+  // Tickets that meet all the filtering criteria requested by the pool.
```

Comment:
why satisfy -> meet?

Reply:
this is what you have in the queryid issue.

Reply:
oh, ok me fighting with the wording I gave.